lib/project_types/script/script_project.rb

```diff
@@ -21,10 +21,9 @@ def lookup_config(key)
     end
 
     class << self
-      def create(dir)
+      def create(ctx, dir)
         raise Errors::ScriptProjectAlreadyExistsError, dir if Dir.exist?(dir)
-
-        FileUtils.mkdir_p(dir)
+        ctx.mkdir_p(dir)
         Dir.chdir(dir)
```

Comment:
Could we swap this with your new `ctx.chdir` method too? We would need to get rid of [this line](https://github.com/Shopify/shopify-app-cli/pull/689/files#diff-cd5d39939d19dc625e6d60c1bad79ab9R22) too if we do that. I think there are a few other places that we do that too

Reply:
This is giving me some trouble, so I'll leave this PR as a WIP and grab some help tomorrow!